reference/7.1/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -337,10 +337,18 @@ Converts or limits objects to the specified type. If the objects cannot be
 converted, PowerShell generates an error.
 
 ```powershell
-[DateTime]"2/20/88" - [DateTime]"1/20/88"
-[Int] (7/2)
-[String] 1 + 0
-[Int] '1' + 0
+# Dates
+[DateTime] '2/20/88' - [DateTime] '1/20/88' -EQ [TimeSpan] '31'
+# Bankers’ rounding
+[Int] (7/2) -EQ 4 -AND [Int] (9/2) -EQ 4
+# Concatenation
+[String] 1 + 0 -EQ '10'
+# Addition
+[Int] '1' + 0 -EQ 1
```

Comment:
I don't think we need these examples. They are just more of the same information. This is also covered in about_Arithmetic_Operators, which is a better place for that discussion.